### PR TITLE
Posixly correct

### PR DESCRIPTION
**Pull request by phadej**
New PR as advised in https://github.com/gruntjs/grunt-contrib-compass/issues/201#issuecomment-72146076

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -87,6 +87,12 @@ module.exports = function (grunt) {
       }
     },
 
+    shell: {
+      posixlyCorrect: {
+        command: "POSIXLY_CORRECT=1 grunt compass:compile"
```
Comment (sindresorhus) [inline Gruntfile.js]: single-quotes

File: tasks/lib/compass.js [hunk 2]
```diff
@@ -169,11 +167,35 @@ exports.init = function (grunt) {
...
+    ]));
+
+    if (grunt.option('no-color')) {
+      args.push('--boring');
+    }
+
+    var doubleDash = false;
+    function pushDoubleDash() {
+      if (!doubleDash) {
+        args.push("--");
```
Comment (sindresorhus) [inline tasks/lib/compass.js]: single-quotes

File: tasks/lib/compass.js [hunk 3]
```diff
@@ -169,11 +167,35 @@ exports.init = function (grunt) {
...
+      args.push('--boring');
+    }
+
+    var doubleDash = false;
+    function pushDoubleDash() {
+      if (!doubleDash) {
+        args.push('--');
+        doubleDash = true;
+      }
+    }
```
Comment (sindresorhus) [inline tasks/lib/compass.js]: Would be cleaner to wrap this function in [`onetime`](https://github.com/sindresorhus/onetime).

**Comment by sindresorhus** [comment]
ping @phadej

**Comment by arthurvr** [comment]
Friendly ping @phadej ;)

**Comment by phadej** [comment]
Sorry for the delay, fixed & updated the PR.

**Pull request closed without merging**